layer/tile/Bing.js

```diff
@@ -47,14 +60,19 @@ L.BingLayer = L.TileLayer.extend({
 			e.parentNode.removeChild(e);
 			if (meta.errorDetails) {
 				throw new Error(meta.errorDetails);
-				return;
 			}
 			_this.initMetadata(meta);
 		};
-		var urlScheme = (document.location.protocol === 'file:') ? 'http' : document.location.protocol.slice(0, -1);
-		var url = urlScheme + '://dev.virtualearth.net/REST/v1/Imagery/Metadata/'
-					+ this.options.type + '?include=ImageryProviders&jsonp=' + cbid +
-					'&key=' + this._key + '&UriScheme=' + urlScheme;
+		var urlScheme = document.location.protocol === 'file:' ? 'http' :
+			document.location.protocol.slice(0, -1);
+		var url = urlScheme + '://dev.virtualearth.net/REST/v1/Imagery/Metadata/' + this.options.type;
```

Comment:
we should ditch all this `urlScheme`things and simply use `htts` every time

Reply:
I heard that some browsers forbid mixed content, if so - it may brake maps on http pages.

Reply:
no, miwed content is only when http content is loaded from http webpage ;)

see https://github.com/leaflet-extras/leaflet-providers/pull/237

Reply:
Ok, I will fix it.